astroplan/constraints.py

```diff
@@ -997,24 +997,59 @@ def observability_table(constraints, observer, targets, times=None,
     return tab
 
 
-def _rescale_minmax(vals, min_val, max_val):
-    """ Rescale altitude into an observability score."""
-    rescaled = (vals - min_val) / (max_val - min_val)
-    below = rescaled < 0
-    above = rescaled > 1
-    rescaled[below] = 0
-    rescaled[above] = 1
+def _rescale_minmax(vals, worst_val, best_val, better_than=1, worse_than=0):
+    """
+    rescales the input ``vals`` between 0 and 1
 
-    return rescaled
+    Parameters
+    ----------
+    vals : array of values
+        the values that need to be rescaled to be between 0 and 1
+    worst_val : value
+        worst acceptable value (rescales to 0)
+    best_val : value
+        best value cared about (rescales to 1)
+    better_than : 0 or 1
+        what is returned for ``vals`` beyond the ``best_val``
+    worse_than : 0 or 1
+        what is returned for ``vals`` beyond the ``worst_val``
 
+    Returns
+    -------
+    array of floats between 0 and 1 inclusive rescaled so that
+    ``vals`` equal to ``worst_val`` equal 0 and those equal to
+    ``best_val`` equal 1
 
-def _rescale_airmass(vals, min_val, max_val):
-    """ Rescale airmass into an observability score."""
-    rescaled = (vals - min_val) / (max_val - min_val)
-    below = rescaled < 0
-    above = rescaled > 1
-    # In both cases, we want out-of-range airmasses to return a 0 score
-    rescaled[below] = 1
-    rescaled[above] = 1
+    Examples
+    --------
+    rescale altitude = [20, 30, 40, 45, 55, 70] to between
+    0 and 1, with the best at 60->1 and worst at 35->0. 0
+    below 35 and 1 above 60.
+    >>> from astroplan.constraints import _rescale_minmax
+    >>> import numpy as np
+    >>> airmasses = np.array([20, 30, 40, 45, 55, 70])
```

Comment:
I think you mean altitudes rather than airmasses here (and in the line below). Also – would you use a complete sentence in the description above, without repeating the variable definitions verbatim?